src/_common/QuotePart/QuotePart.BufferList.cs

```diff
@@ -37,15 +37,15 @@ public void Add(IReadOnlyList<IQuote> quotes)
         }
     }
 
-    /// <summary>
+    /// <inheritdoc />
```

Comment:
You've messed these up in a fairly bad way.  Is this supposed to be "inheritdoc" or the opening element of the "summary" statement.